packages/app/src/app/overmind/effects/gql/dashboard/fragments.ts

```diff
@@ -213,3 +213,20 @@ export const npmRegistryFragment = gql`
     teamId
   }
 `;
+
+export const branchFragment = gql`
+  fragment branchFragment on Branch {
```

Comment:
This is minor, but I would call it just `fragment branch ...` because this causes the type generated to be `BranchFragmentFragment`: https://github.com/codesandbox/codesandbox-client/pull/6915/files#diff-b002ac4b449079f158a3322d399854374913f525d311519f39d245be57e4963aR2287